pages/guides/containers.md

```diff
@@ -126,6 +154,9 @@ Also pay close attention to the `-p` parameter above:
 * We used ports 3000:3001 but you need to adapt them to your web framework
   (such as 4000:4001 for Phoenix).
 
+* We forward the ports for the Tidewave CLI as well, such that you can access it
+  at `http://localhost:9832`.
```

Comment:
If you have the desktop app, then it will conflict, right? We probably need to pick another port for the docker one.